test/boost/view_schema_pkey_test.cc

```diff
@@ -715,6 +715,12 @@ SEASTAR_TEST_CASE(test_base_non_pk_columns_in_view_partition_key_are_non_emtpy)
                     });
         }
 
+        // The following if'ed-out tests verified behavior that we
+        // no longer believe to be correct, and also turns out not to be
+        // compatible with Cassandra (see issue #9375): these tests checked
+        // that if a view row should get an empty string as a partition key,
+        // this row should not be generated. But this is not the case.
+#if 0
```

Comment:
delete?

Reply:
Ok. I thought maybe it has some historic significance, but yes, git can fulfill that role.